applications/rag/variables.tf

```diff
@@ -366,3 +366,9 @@ variable "goog_cm_deployment_name" {
   type    = string
   default = ""
 }
+
+variable "disable_ray_cluster_network_policy" {
```

Comment:
If we update here, do we need to update application/ray?

Reply:
I didn't bother to update it in `application/ray` since we want that to have networkpolicy by default.

Reply:
I'd like to see us work through the implications of this in the Ray application. gmp-operator should be made more resilient as opposed to discarding security. We can always turn it off in the next commit if it's too much of a hassle.